heisenbridge/control_room.py

```diff
@@ -570,19 +570,19 @@ async def cmd_media_url(self, args):
         if args.remove:
             self.serv.config["media_url"] = None
             await self.serv.save()
-            self.serv.endpoint = await self.serv.detect_public_endpoint()
+            self.serv.media_endpoint = await self.serv.detect_public_endpoint()
```

Comment:
This should probably reset it to the one set in the registration (or `None` if that's not set either)